core/pom.xml

```diff
@@ -76,11 +76,22 @@
         </dependency>
 
         <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+        <dependency>
             <groupId>org.json</groupId>
             <artifactId>json</artifactId>
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
+            <version>1.13</version>
+        </dependency>
```

Comment:
Already using it with Android, e.g. GraphHopper Android [sample](https://github.com/graphhopper/graphhopper/blob/master/android/app/build.gradle#L52).